book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rents
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rents = []
+  end
+end
```

Comment:
- [ ] Your Book class should have a  method for adding rentals, it look something like this :point_down: 

```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```